packages/editor-ui/src/components/helpers.ts

```diff
@@ -67,3 +68,9 @@ export function isString(value: unknown): value is string {
 export function isNumber(value: unknown): value is number {
 	return typeof value === 'number';
 }
+
+// Single source of truth for the credentials fields position
+// This method should be used whenever default value is needed
+export function getDefaultCredentialsIndex(node: INode): number {
+	return node.type === WEBHOOK_NODE_TYPE ? 1 : 0;
+}
```

Comment:
this works... but we could do better. we can try to look through the params and credentials dependency and figure out where the credential should set..
because there's exceptions we need to look at here 
- Git node
- Pipedrive
- Sentry io